src/Microsoft.IdentityModel.Tokens/Base64UrlEncoder.cs

```diff
@@ -254,12 +223,8 @@ internal static unsafe byte[] UnsafeDecode(string str)
                     fixed (char* src = str)
                     fixed (char* dest = decodedString)
                     {
-#if NET45
-                        for (int index = 0; index < str.Length; index++)
-                            dest[index] = src[index];
-#else
                         Buffer.MemoryCopy(src, dest, str.Length * 2, str.Length * 2);
-#endif
+
```

Comment:
extra line?

Reply:
I don't know if it's better style to have an empty line between the memcpy and the other lines. I can remove it.